security/pkg/server/ca/authenticate/kubeauth/kube_jwt.go

```
@@ -101,6 +102,9 @@ func (a *KubeJWTAuthenticator) authenticateHTTP(req *http.Request) (*security.Ca
 		return nil, fmt.Errorf("target JWT extraction error: %v", err)
 	}
 	clusterID := cluster.ID(req.Header.Get(clusterIDMeta))
+	if !features.CentralIstiodAccess {
```

Should we reject instead of silently drop it?